examples/tfengine/README.md

```diff
@@ -1,7 +1,9 @@
-# Generated Terraform configs from Engine files
+# Example Terraform Engine configs
 
-This directory contains Terraform configs generated by the Terraform Engine
-using example configs.
+This directory contains example Terraform Engine configs.
+
+The [generated/](./generated) contains Terraform configs generated by the
```

Comment:
Typo, should be "the generated/ **directory**" or "dir"

Reply:
Good catch. Done.